ansible/ci-build-collector.yml

```diff
@@ -123,11 +109,9 @@
         msg:
         - "Pushed the following images:"
         - "  {{ collector_image }}-{{ arch }}"
-        - "  {{ collector_image }}-{{ arch }}-slim"
         - "  {{ collector_image }}-{{ arch }}-base"
         - "  {{ collector_image }}-{{ arch }}-latest"
```

Comment:
Not a blocking comment and not aiming to be delay the PR by becoming a reviewer.

I recalled this our conversation https://redhat-internal.slack.com/archives/CFMQ5C2TT/p1733309587336659

Your changes in https://github.com/stackrox/stackrox/pull/13350/files#diff-9f1c5124b529a407693ba184291c9797252978fe4ce8770e1ddec20d811efd69L104 allow getting rid of publishing `-latest`.

Also, I believe `-base` wasn't used by StackRox. IIRC, it was created by the Collector build pipelines as something intermediate before drivers are injected and it becomes "the full collector".